e2e/scenario_helpers_test.go

```diff
@@ -135,7 +141,13 @@ func prepareAKSNode(ctx context.Context, s *Scenario) {
 	}
 
 	require.NoError(s.T, err)
-	createVMSS(ctx, s)
+
+	attachVMIdentity := false
```

Comment:
can just simplify this to `createVMSS(ctx, s, s.Tags.NonAnonymousACR)`

Reply:
actually, does that mean we can just resolve the bool directly within `createVMSS` and leave off the extra parameter?